<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="4" ignored="0" total="5" passed="1">
  <reporter-output>
    <line>
      <![CDATA[<a href='D:\PrasenjitOffice\OneDrive - Kovair Software\My Kovair Files\EclipseWorkspace\JenkinsIntegration1\src\test\java\scrpackage\NGTest1954.png'> <img src='D:\PrasenjitOffice\OneDrive - Kovair Software\My Kovair Files\EclipseWorkspace\JenkinsIntegration1\src\test\java\scrpackage\NGTest1954.png' height='100' width='100'/> </a>]]>
    </line>
    <line>
      <![CDATA[<a href='D:\PrasenjitOffice\OneDrive - Kovair Software\My Kovair Files\EclipseWorkspace\JenkinsIntegration1\src\test\java\scrpackage\NGTest2937.png'> <img src='D:\PrasenjitOffice\OneDrive - Kovair Software\My Kovair Files\EclipseWorkspace\JenkinsIntegration1\src\test\java\scrpackage\NGTest2937.png' height='100' width='100'/> </a>]]>
    </line>
    <line>
      <![CDATA[<a href='D:\PrasenjitOffice\OneDrive - Kovair Software\My Kovair Files\EclipseWorkspace\JenkinsIntegration1\src\test\java\scrpackage\NGTest3232.png'> <img src='D:\PrasenjitOffice\OneDrive - Kovair Software\My Kovair Files\EclipseWorkspace\JenkinsIntegration1\src\test\java\scrpackage\NGTest3232.png' height='100' width='100'/> </a>]]>
    </line>
    <line>
      <![CDATA[<a href='D:\PrasenjitOffice\OneDrive - Kovair Software\My Kovair Files\EclipseWorkspace\JenkinsIntegration1\src\test\java\scrpackage\NGTest4247.png'> <img src='D:\PrasenjitOffice\OneDrive - Kovair Software\My Kovair Files\EclipseWorkspace\JenkinsIntegration1\src\test\java\scrpackage\NGTest4247.png' height='100' width='100'/> </a>]]>
    </line>
  </reporter-output>
  <suite name="Suite" duration-ms="30551" started-at="2019-06-13T06:09:11Z" finished-at="2019-06-13T06:09:41Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="30551" started-at="2019-06-13T06:09:11Z" finished-at="2019-06-13T06:09:41Z">
      <class name="kovair.JenkinsIntegration1.JenkinsIntegrationTestNGClass">
        <test-method status="PASS" signature="BeforeTest()[pri:0, instance:kovair.JenkinsIntegration1.JenkinsIntegrationTestNGClass@57baeedf]" name="BeforeTest" is-config="true" duration-ms="27500" started-at="2019-06-13T06:09:11Z" finished-at="2019-06-13T06:09:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeTest -->
        <test-method status="FAIL" signature="NGTest1()[pri:0, instance:kovair.JenkinsIntegration1.JenkinsIntegrationTestNGClass@57baeedf]" name="NGTest1" duration-ms="12" started-at="2019-06-13T06:09:38Z" finished-at="2019-06-13T06:09:38Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at kovair.JenkinsIntegration1.JenkinsIntegrationTestNGClass.NGTest1(JenkinsIntegrationTestNGClass.java:41)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- NGTest1 -->
        <test-method status="PASS" signature="AfterMethod(org.testng.ITestResult)[pri:0, instance:kovair.JenkinsIntegration1.JenkinsIntegrationTestNGClass@57baeedf]" name="AfterMethod" is-config="true" duration-ms="566" started-at="2019-06-13T06:09:38Z" finished-at="2019-06-13T06:09:39Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=NGTest1 status=FAILURE method=JenkinsIntegrationTestNGClass.NGTest1()[pri:0, instance:kovair.JenkinsIntegration1.JenkinsIntegrationTestNGClass@57baeedf] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[<a href='D:\PrasenjitOffice\OneDrive - Kovair Software\My Kovair Files\EclipseWorkspace\JenkinsIntegration1\src\test\java\scrpackage\NGTest1954.png'> <img src='D:\PrasenjitOffice\OneDrive - Kovair Software\My Kovair Files\EclipseWorkspace\JenkinsIntegration1\src\test\java\scrpackage\NGTest1954.png' height='100' width='100'/> </a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- AfterMethod -->
        <test-method status="FAIL" signature="NGTest2()[pri:0, instance:kovair.JenkinsIntegration1.JenkinsIntegrationTestNGClass@57baeedf]" name="NGTest2" duration-ms="2" started-at="2019-06-13T06:09:39Z" finished-at="2019-06-13T06:09:39Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at kovair.JenkinsIntegration1.JenkinsIntegrationTestNGClass.NGTest2(JenkinsIntegrationTestNGClass.java:49)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- NGTest2 -->
        <test-method status="PASS" signature="AfterMethod(org.testng.ITestResult)[pri:0, instance:kovair.JenkinsIntegration1.JenkinsIntegrationTestNGClass@57baeedf]" name="AfterMethod" is-config="true" duration-ms="393" started-at="2019-06-13T06:09:39Z" finished-at="2019-06-13T06:09:39Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=NGTest2 status=FAILURE method=JenkinsIntegrationTestNGClass.NGTest2()[pri:0, instance:kovair.JenkinsIntegration1.JenkinsIntegrationTestNGClass@57baeedf] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[<a href='D:\PrasenjitOffice\OneDrive - Kovair Software\My Kovair Files\EclipseWorkspace\JenkinsIntegration1\src\test\java\scrpackage\NGTest2937.png'> <img src='D:\PrasenjitOffice\OneDrive - Kovair Software\My Kovair Files\EclipseWorkspace\JenkinsIntegration1\src\test\java\scrpackage\NGTest2937.png' height='100' width='100'/> </a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- AfterMethod -->
        <test-method status="FAIL" signature="NGTest3()[pri:0, instance:kovair.JenkinsIntegration1.JenkinsIntegrationTestNGClass@57baeedf]" name="NGTest3" duration-ms="2" started-at="2019-06-13T06:09:39Z" finished-at="2019-06-13T06:09:39Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at kovair.JenkinsIntegration1.JenkinsIntegrationTestNGClass.NGTest3(JenkinsIntegrationTestNGClass.java:59)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- NGTest3 -->
        <test-method status="PASS" signature="AfterMethod(org.testng.ITestResult)[pri:0, instance:kovair.JenkinsIntegration1.JenkinsIntegrationTestNGClass@57baeedf]" name="AfterMethod" is-config="true" duration-ms="393" started-at="2019-06-13T06:09:39Z" finished-at="2019-06-13T06:09:40Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=NGTest3 status=FAILURE method=JenkinsIntegrationTestNGClass.NGTest3()[pri:0, instance:kovair.JenkinsIntegration1.JenkinsIntegrationTestNGClass@57baeedf] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[<a href='D:\PrasenjitOffice\OneDrive - Kovair Software\My Kovair Files\EclipseWorkspace\JenkinsIntegration1\src\test\java\scrpackage\NGTest3232.png'> <img src='D:\PrasenjitOffice\OneDrive - Kovair Software\My Kovair Files\EclipseWorkspace\JenkinsIntegration1\src\test\java\scrpackage\NGTest3232.png' height='100' width='100'/> </a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- AfterMethod -->
        <test-method status="FAIL" signature="NGTest4()[pri:0, instance:kovair.JenkinsIntegration1.JenkinsIntegrationTestNGClass@57baeedf]" name="NGTest4" duration-ms="3" started-at="2019-06-13T06:09:40Z" finished-at="2019-06-13T06:09:40Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at kovair.JenkinsIntegration1.JenkinsIntegrationTestNGClass.NGTest4(JenkinsIntegrationTestNGClass.java:66)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- NGTest4 -->
        <test-method status="PASS" signature="AfterMethod(org.testng.ITestResult)[pri:0, instance:kovair.JenkinsIntegration1.JenkinsIntegrationTestNGClass@57baeedf]" name="AfterMethod" is-config="true" duration-ms="325" started-at="2019-06-13T06:09:40Z" finished-at="2019-06-13T06:09:40Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=NGTest4 status=FAILURE method=JenkinsIntegrationTestNGClass.NGTest4()[pri:0, instance:kovair.JenkinsIntegration1.JenkinsIntegrationTestNGClass@57baeedf] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[<a href='D:\PrasenjitOffice\OneDrive - Kovair Software\My Kovair Files\EclipseWorkspace\JenkinsIntegration1\src\test\java\scrpackage\NGTest4247.png'> <img src='D:\PrasenjitOffice\OneDrive - Kovair Software\My Kovair Files\EclipseWorkspace\JenkinsIntegration1\src\test\java\scrpackage\NGTest4247.png' height='100' width='100'/> </a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- AfterMethod -->
        <test-method status="PASS" signature="NGTest5()[pri:0, instance:kovair.JenkinsIntegration1.JenkinsIntegrationTestNGClass@57baeedf]" name="NGTest5" duration-ms="1" started-at="2019-06-13T06:09:40Z" finished-at="2019-06-13T06:09:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- NGTest5 -->
        <test-method status="PASS" signature="AfterMethod(org.testng.ITestResult)[pri:0, instance:kovair.JenkinsIntegration1.JenkinsIntegrationTestNGClass@57baeedf]" name="AfterMethod" is-config="true" duration-ms="1" started-at="2019-06-13T06:09:40Z" finished-at="2019-06-13T06:09:40Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=NGTest5 status=SUCCESS method=JenkinsIntegrationTestNGClass.NGTest5()[pri:0, instance:kovair.JenkinsIntegration1.JenkinsIntegrationTestNGClass@57baeedf] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterMethod -->
        <test-method status="PASS" signature="AfterTest()[pri:0, instance:kovair.JenkinsIntegration1.JenkinsIntegrationTestNGClass@57baeedf]" name="AfterTest" is-config="true" duration-ms="1254" started-at="2019-06-13T06:09:40Z" finished-at="2019-06-13T06:09:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterTest -->
      </class> <!-- kovair.JenkinsIntegration1.JenkinsIntegrationTestNGClass -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
